default.nix

```diff
@@ -0,0 +1,12 @@
+let
+  nix-pre-commit-hooks = import (builtins.fetchTarball "https://github.com/cachix/pre-commit-hooks.nix/tarball/master");
+in {
+  pre-commit-check = nix-pre-commit-hooks.run {
```

Comment:
@raulraja I want you to take a look at this beauty: on CI, run ormolu, elm-format, htlint... all on pre-commit and on CI for free with Nix! ❄️ 😍